reviews/serializers.py

```diff
@@ -60,5 +83,44 @@ class ReviewLogSerializer(JSONAPISerializer):
     def get_log_url(self, obj):
         return absolute_reverse('reviews:review_log-detail', kwargs={'log_id': obj._id, 'version': self.context['request'].parser_context['kwargs']['version']})
 
+    def create(self, validated_data):
+        action = validated_data.pop('action')
+        user = validated_data.pop('user')
+        reviewable = validated_data.pop('reviewable')
+        comment = validated_data.pop('comment', '')
+
+        try:
+            if action == Actions.ACCEPT.value:
```

Comment:
Yeah, I don't totally like what I did, but it leaves no room for confusion. I don't have a solid enough idea of what workflows will look like in the future to make something we'll continue to be happy with, so I think we should stick with simple for now.